c7n/mu.py

```diff
@@ -63,8 +63,11 @@ class PythonPackageArchive(object):
 
     zip_compression = zipfile.ZIP_DEFLATED
 
-    def __init__(self, *modules):
+    def __init__(self, *modules, cache_file=None):
```

Comment:
@kapilt is there a reason you use `*modules` instead of `modules=[]`?

I see that `c7n` passes array, `c7n_mailer` passes multiple arguments.

Reply:
no particular reason, its something we can fix, the primary interface for custodian is via custodian_archive, we have a few callers that call it directly but their all in tree and there's isn't a stable abi for external consumers.